mosh_app/mosh_window.js

```diff
@@ -36,7 +36,14 @@ function execMosh() {
 
   // Don't exit fullscreen with ESC.
   terminal.document_.onkeyup = function(e) {
-    if (e.keyCode == 27) {
+    if (e.keyCode == 78 && e.ctrlKey && e.shiftKey){
```

Comment:
this isn't the right way to bind this.  use `terminal.keyboard.bindings.addBinding` inside of `terminal.onTerminalReady` instead.

example: https://chromium-review.googlesource.com/389492

Reply:
i simply stuck with the pattern from the existing code. if y'all want to overhaul the key bindings fully, that's beyond the scope of this change.

Reply:
that's simply not true.  the ESC key hack here is not doable in hterm and is to workaround fullscreen behavior on some setups.  that's completely different from adding new keybindings.